.github/workflows/error-codes-updater.yml

```diff
@@ -0,0 +1,63 @@
+name: Meshsync Error Codes Utility Runner
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - '**.go'
+
+jobs:
+  Update-error-codes:
+    name: Error codes utility
+    if: github.repository == 'layer5io/meshsync'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # token here with write access to meshsync repo
+        with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ secrets.GO_VERSION }}
+
+      - name: Run utility
+        run: |
+          go run github.com/layer5io/meshkit/cmd/errorutil -d . update --skip-dirs meshery -i ./helpers -o ./helpers
+      # to update errorutil* files in meshkit repo
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_user_name: l5io
+          commit_user_email: ci@layer5.io
+          commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          commit_options: '--signoff'
+          commit_message: "run error codes utility"
+          file_pattern: helpers/ **.go
+
+      # to push changes to meshery docs
+      - name: Checkout meshery
+        uses: actions/checkout@v2
+        with:
+          repository: 'meshery/meshery'
+          # token with write access to meshery repository
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          path: 'meshery'
+          ref: 'master'
+
+      - name: Update and push docs
+        run: |
+          echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
+          cd ./meshery
+          git config user.name l5io
+          git config user.email ci@layer5.io
+          if git diff --exit-code --quiet
+          then
+            echo "No changes to commit"
+            exit
+          fi
+          git add ./docs/_data/errorref/meshkit_errors_export.json
```

Comment:
```suggestion
          git add ./docs/_data/errorref/meshsync_errors_export.json
```

Reply:
In these, I let it remain as meshkit errors thinking that's describing the style of errors used(meshkit errors), not the repo.
But I think you're right here